matrix-sdk-android/src/main/java/im/vector/matrix/android/internal/session/room/RoomAPI.kt

```diff
@@ -129,6 +129,21 @@ internal interface RoomAPI {
              @Body content: Content?
     ): Call<SendResponse>
 
+    /**
+     * Send an event to a room.
+     *
+     * @param txId      the transaction Id
+     * @param roomId    the room id
+     * @param eventType the event type
+     * @param content   the event content as string
+     */
+    @PUT(NetworkConstants.URI_API_PREFIX_PATH_R0 + "rooms/{roomId}/send/{eventType}/{txId}")
+    fun send(@Path("txId") txId: String,
+             @Path("roomId") roomId: String,
+             @Path("eventType") eventType: String,
+             @Body content: String?
```

Comment:
Nullable?

Reply:
I kept the same than with plain object, but in practice it won't be null

Reply:
ok